src/commands/app-configs/deploy.ts

```diff
@@ -97,4 +98,16 @@ the application configuration was created.
     const data = result.data as any
     return data.deployments
   }
+
+  // Show a warning if a user tries to deploy an app-config when it is already deployed
+  handleError(error: MixError) {
+    debug('handleError() error.statusCode: %d', error.statusCode)
+
+    if (error.statusCode === 400 && error.message.toLocaleLowerCase().includes('deployment already completed')) {
+      this.warn(`Application configuration was already deployed. 
+It is not possible to re-deploy an already deployed application configuration`)
```

Comment:
Add a period at the end of the sentence for consistency with the first sentence.